.github/workflows/try-setup-install.yml

```diff
@@ -0,0 +1,37 @@
+name: Try Setup Install
+
+on: [push]
+
+jobs:
+
+  gather-py3-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.py3_versions.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Python Versions
+        id: py3_versions
+        run: |
+          minmin_maxmax=$(grep -P "\(\(\d+, ?\d+\), ?\(\d+, ?\d+\)\)" -oh setup.py | sed 's/[^0-9]/ /g')
+          IFS=', ' read -r -a array <<< "$minmin_maxmax"
+          min_thru_max_series=$(for i in `seq ${array[1]} ${array[3]}`; do printf "3.$i,"; done | rev | cut -c 2- | rev)
```

Comment:
I think this `cut -c 2-` might only allow 3.X

Where I've tried to specify (3,10) with LTA, it is trying to run setup with Python 3.1 ... :open_mouth:

I can bump LTA down to (3,9), but I thought you'd like a heads-up bug report.

Reply:
it was being interpreted at a number at some point. Adding quotes fixed this

Reply:
cut removes a trailing comma